test/core/end2end/tests/retry_streaming.cc

```diff
@@ -384,6 +395,11 @@ static void test_retry_streaming(grpc_end2end_test_config config) {
   GPR_ASSERT(0 == call_details.flags);
   GPR_ASSERT(was_cancelled == 1);
 
+  GPR_ASSERT(channelz_channel != nullptr);
+  char* json = channelz_channel->RenderJsonString();
+  GPR_ASSERT(json != nullptr);
+  gpr_log(GPR_INFO, "%s", json);
```

Comment:
The test should actually verify that the right entries are logged when the resolver returns.

Reply:
Done